doc/optimizers.md

```diff
@@ -2107,7 +2167,7 @@ size equal to that of the starting population.
 #### Constructors
 
  * `NSGA2()`
- * `NSGA2(`_`populationSize, maxGenerations, crossoverProb, mutationProb, mutationStrength, epsilon, lowerBound, upperBound`_`)`
```

Comment:
Why remove this parameter from the documentation?  It's still in the code (and the table below).

Reply:
Sorry for the mistake reverted it